packages/backend/src/handlers/triggers/updateDiscordRole.ts

```diff
@@ -17,73 +18,77 @@ export const updateDiscordRole = async (
 ) => {
   const { old: oldPlayer, new: newPlayer } = payload.event.data;
 
+  console.log(`updateDiscordRole action triggered for player (username=${newPlayer?.username})`);
+
   try {
     if (newPlayer == null) return;
    
     const getPlayerResponse = await client.GetPlayer({ playerId: newPlayer.id });
-    const discordPlayerAccount = getPlayerResponse.player_by_pk?.Accounts?.find(a => a.type === AccountType_Enum.Discord);
-    if (discordPlayerAccount?.identifier == null) return;
+    const playerDiscordId = getPlayerResponse.player_by_pk?.discord_id;
+    if (playerDiscordId == null) return;
```

Comment:
```suggestion
    if (!playerDiscordId) return;
```
in case `getPlayerResponse.player_by_pk?.discord_id` returns undefined

Reply:
He got me with that one too, `undefined == null` is `true`.